src/helpers/App.php

```diff
@@ -853,7 +853,8 @@ public static function projectConfigConfig(): array
     {
         return [
             'class' => ProjectConfigService::class,
-            'readOnly' => Craft::$app->getIsInstalled() && !Craft::$app->getConfig()->getGeneral()->allowAdminChanges,
+            'readOnly' => self::isEphemeral() || (Craft::$app->getIsInstalled() && !Craft::$app->getConfig()->getGeneral()->allowAdminChanges),
+            'writeYamlAutomatically' => !self::isEphemeral()
         ];
     }
 
```

Comment:
I'm not 💯 clear on the interaction between `readOnly` and `writeYamlAutomatically`…

Either way, we don't ever want it to write when `isEphermeral`, so toggling both.